lib/ui/semantics.dart

```diff
@@ -475,10 +476,10 @@ class SemanticsFlag {
   /// Whether the platform can scroll the semantics node when the user attempts
   /// to move focus to an offscreen child.
   ///
-  /// For example, a [ListView] widget has implicit scrolling so that users can
-  /// easily move to the next visible set of children. A [TabBar] widget does
-  /// not have implicit scrolling, so that users can navigate into the tab
-  /// body when reaching the end of the tab bar.
+  // For example, a [ListView] widget has implicit scrolling so that users can
```

Comment:
This one needs three ///

Reply:
Gah -- that's what I get for updating the C++ side first ;) Done.